data-raw/make_cluster_vcov.R

```diff
@@ -0,0 +1,50 @@
+
+# Create clustered variance-covariance matrix for package tests
+
+library(modmarg)
+library(sandwich)
+library(data.table)
+
```

Comment:
Can we call functions from those packages instead of loading the entire namespace?